### PR TITLE
feat: Add table format

### PR DESCRIPTION
**Pull request by AlasDiablo**

### CODE REVIEW

**Review by jj618 — APPROVED** [review]
LGTM

File: src/app/custom/translations.tsv [hunk 1]
```diff
@@ -558,14 +558,16 @@
...
-"type"	"type"	"type"
-"number_of_column"	"number_of_column"	"number_of_column"
+"table_title"	"Title (title)"	"Titre (title)"
+"table_summary"	"Summary (summary)"	"Description (summary)"
+"table_source"	"Source (source)"	"Source (source)"
+"table_target"	"Target (target)"	"Cible (target)"
+"table_weight"	"Weight (weight)"	"Poids (weight)"
+"table_value"	"Value (value)"	"Valeur (value)"
+"table_url"	"Url (url)"	"Url (url)"
+"table_id"	"Identifier (_id)"	"Identifient (_id)"
```
Comment (parmentf) [inline src/app/custom/translations.tsv]:
```suggestion
"table_id"	"Identifier (_id)"	"Identifiant (_id)"
```

File: src/app/custom/translations.tsv [hunk 2]
```diff
@@ -558,14 +558,16 @@
...
+"table_other"	"Custom field"	"Champ personalisé"
```
Comment (parmentf) [inline src/app/custom/translations.tsv]:
```suggestion
"table_other"	"Custom field"	"Champ personnalisé"
```

File: src/app/custom/translations.tsv [hunk 3]
```diff
@@ -558,14 +558,16 @@
...
+"table_other_field"	"Custom field name"	"Nom du champ personalisé"
```
Comment (parmentf) [inline src/app/custom/translations.tsv]:
```suggestion
"table_other_field"	"Custom field name"	"Nom du champ personnalisé"
```

File: src/app/custom/translations.tsv [hunk 4]
```diff
@@ -558,14 +558,16 @@
...
+"table_field"	"Routine field (name)"	"Champ de la routine (nom)"
+"column_title"	"Column title"	"Titre de la colonne"
+"number_of_column"	"Number of column"	"Nombre de colonne"
```
Comment (parmentf) [inline src/app/custom/translations.tsv]:
```suggestion
"number_of_column"	"Number of columns"	"Nombre de colonnes"
```

File: src/app/js/formats/table/TableView.js [hunk 5]
```diff
@@ -17,31 +17,60 @@ import {
...
-            page: this.state.page,
+        });
+    }
+
+    sort(id) {
+        let sort = this.state.sort;
+        if (id === this.state.sortId) {
+            switch (sort) {
+                case 'asc':
+                    sort = 'desc';
```
Comment (jj618) [inline src/app/js/formats/table/TableView.js:58]: Remove case false

File: src/app/js/formats/table/TableView.js [hunk 6]
```diff
@@ -77,20 +106,55 @@ class TableView extends Component {
             );
         };
```
Comment (jj618) [inline src/app/js/formats/table/TableView.js:105]: rename id ot columnId
if(id !== this.state.sortId) return false;

return this.state.sort;

**Review by jj618 — APPROVED** [review]
LGTM

**Review by jj618 — APPROVED** [review]

**Pull request merged**